torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -623,6 +623,16 @@ int64_t AtenXlaType::size(const at::Tensor& self, int64_t dim) const {
   return bridge::GetXlaTensor(self).size(dim);
 }
 
+at::Tensor AtenXlaType::embedding(const at::Tensor& weight,
+                                  const at::Tensor& indices,
+                                  int64_t padding_idx, bool scale_grad_by_freq,
+                                  bool sparse) const {
+  // TODO: for now route to native, which dispatches supported XLA operations.
+  // We need to make use of the TPU embedding core here eventually.
+  return at::native::embedding(weight, indices, padding_idx, scale_grad_by_freq,
```

Comment:
What's the reason for not using `AtenXlaTypeBase::embedding`?

Reply:
Same reason why we switched yesterday to native:: for einsum(). It is composed by operations we already support.

Reply:
Oh, that's nice.